tests/console/yast2_cmdline.pm

```diff
@@ -40,8 +42,22 @@ sub run_yast_cli_test {
 }
 
 sub run {
+    my $cmd;
+
     select_console 'root-console';
 
+    if (is_sle) {
+        if (sle_version_at_least('15') and get_var('REPO_SLE15_MODULE_BASESYSTEM_SOURCE')) {
+            $cmd = "ar -f " . "$utils::OPENQA_FTP_URL/" . get_var('REPO_SLE15_MODULE_BASESYSTEM_SOURCE') . " repo-source";
+        }
+        elsif (sle_version_at_least('12-SP4') and get_var('REPO_SLES_SOURCE')) {
+            $cmd = "ar -f " . "$utils::OPENQA_FTP_URL/" . get_var('REPO_SLES_SOURCE') . " repo-source";
+        }
+    }
+
+    zypper_call($cmd) if defined $cmd;
+    zypper_call("ref");
```

Comment:
You already add repos with the parameter `-f` so this call should not be necessary.

I guess the whole above block can be simplified to

```perl
if (is_sle) {
    zypper_call('ar -f ' . $utils::OPENQA_FTP_URL/ . get_var('REPO_SLE15_MODULE_BASESYSTEM_SOURCE', get_required_var('REPO_SLES_SOURCE')) . ' repo-source');
}
```

Reply:
```
 addrepo (ar) [options] FILE.repo
               Add a new repository specified by URI and assign specified alias to it or specify URI to a .repo file. + Newly added repositories have auto-refresh  disabled  by  default
               (except  for  repositories  imported  from  a  .repo,  having  the auto-refresh enabled). To enable auto-refresh use addrepo -f, or the --refresh option of the modifyrepo
               command. + Also, this command does not automatically refresh the newly added repositories. The repositories will get refreshed when used for the first time,  or  you  can
               use the refresh command after finishing your modifications with *repo commands. See also METADATA REFRESH POLICY section for more details.
```
-f just enables autorefresh, without refreshing the added repo.

Reply:
yes, autorefresh. But doesn't that mean that the install call will refresh it?

Reply:
I have tested it, and it seems like the repos are being refreshed when they are used for the first time.